package.json

```diff
@@ -38,7 +38,7 @@
     "eslint-plugin-n": "^15.3.0",
     "eslint-plugin-promise": "^6.0.1",
     "jest": "^29.1.2",
-    "nodemon": "^1.19.4",
+    "nodemon": "^2.0.20",
```

Comment:
Even thought we don't use it, I'd suggest keeping it unchanged.

Reply:
done